dotcom-rendering/makefile

```diff
@@ -30,13 +30,15 @@ deploy:
 
 # prod #########################################
 
-build: clean-dist install
+build: export NODE_ENV=production
+build: clean-dist install gen-dotenv
 	$(call log, "building production bundles")
-	@NODE_ENV=production webpack --config ./scripts/webpack/webpack.config.js --progress
+	webpack --config ./scripts/webpack/webpack.config.js --progress
 
-build-ci: clean-dist install
+build-ci: export NODE_ENV=production
+build-ci: clean-dist install gen-dotenv
 	$(call log, "building production bundles")
-	@NODE_ENV=production SKIP_LEGACY=true webpack --config ./scripts/webpack/webpack.config.js
+	SKIP_LEGACY=true webpack --config ./scripts/webpack/webpack.config.js
```

Comment:
I'm intrigued by this change, could we quickly outline what it does?

I've also noticed all other calls in the makefile start with `@`, but on line 41 and 36 this has been removed?

Reply:
It's using [target specific variable values](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html) so it sets the value for the whole step, and thus substeps, not just the command. 

The removal of the `@` sign was a mistake. It doesn't make a difference to the scripts as it [suppresses echoing](https://www.gnu.org/software/make/manual/make.html#Recipe-Echoing). I've added it back - nice spot!